packages/grafana-flamegraph/src/FlameGraphContainer.test.tsx

```diff
@@ -40,9 +40,13 @@ describe('FlameGraphContainer', () => {
     render(<FlameGraphContainerWithProps />);
     await userEvent.click((await screen.findAllByTitle('Highlight symbol'))[0]);
     expect(screen.getByDisplayValue('net/http.HandlerFunc.ServeHTTP')).toBeInTheDocument();
+    // Unclick the selection so that the rest of the tests don't have it highlighted
```

Comment:
Could update this comment as the clicking/unclicking is isolated to this test. Basically we need to unclick so that we can click something else and continue the test checks.